### PR TITLE
feat: use the better video for database

### PR DESCRIPTION
**Pull request by elliottneilclark**
Summary:
- Use the newly uploaded database video on the database page
- Change the database page to have a better ML section

Test Plan:
- Local testing
- netlify test deploy

### CODE REVIEW

**Review by jfaz1 — APPROVED** [review]

File: static/src/pages/solutions/database.astro [hunk 1]
```diff
@@ -104,7 +102,7 @@ const metadata = {
       {
         title: 'Machine Learning',
         description:
-          'Batteries Included provides an integrated development environment to help your team get machine learning projects off the ground faster, and with better results. Prepare data, host models, and manage pipelines with ease. Our environment makes it easy to integrate machine learning projects into your team’s day-to-day, with space for notebooks, code, data, and more.',
+          'Batteries Included provides an easy to use vector databases and more tools ready at the press of a button. Our environment makes it easy to integrate machine learning projects into your team’s day-to-day, with embedding models, jypter notebooks, and PGVector, while keeping your data local and secure.',
```
Comment (jfaz1) [inline static/src/pages/solutions/database.astro:105]: jypter -> Jupyter

**Review by jfaz1 — DISMISSED** [review]

File: static/src/pages/solutions/database.astro (hunk 1 above)
Comment (jfaz1) [reply static/src/pages/solutions/database.astro:105]: change:
`Batteries Included provides an easy to use vector databases...`
to:
`Batteries Included provides easy to use vector databases`

**Comment by elliottneilclark** [comment]
Today I learned: [Multi-line template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings)

File: static/src/pages/solutions/database.astro
Comment (jfaz1) [inline static/src/pages/solutions/database.astro:1]: Maybe use something like:

```
Batteries Included provides easy-to-use vector databases and a suite of powerful tools, ready at the press of a button.
Our environment makes it easy to build machine learning projects using Ollama hosted models for text embedding generation. 
The Kubernetes-based platform includes easy-to-use Jupyter notebooks with straightforward PGVector integration; keeping your data local and secure.`,
```

Just some minor wording nits.

**Review by JTarasovic — APPROVED** [review]

**Review by jsonmaur — APPROVED** [review]

**Pull request merged**